graphql/config_resolver.go

```diff
@@ -0,0 +1,21 @@
+package graphql
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+// Code generated by github.com/99designs/gqlgen version v0.17.36
+
+import (
+	"context"
+
+	"github.com/evergreen-ci/evergreen/rest/model"
+)
+
```

Comment:
it actually gets generated from the GraphQL filename; so because the value is declared in `config.graphql`, it gets put in `config_resolver.go`. Do you want me to try renaming GraphQL file to something else? 🤔

Reply:
Ah if this is coming from the admin settings then it would make sense; is this just consumed by the distro page?

Reply:
yep thats right, when you have the "Docker" provider it lets you pick from the container pools which are defined in the admin settings:

<img width="346" alt="Screenshot 2023-09-06 at 2 41 46 PM" src="https://github.com/evergreen-ci/evergreen/assets/47064971/cf8722a3-68d2-4cb9-8062-53b16598754a">

Reply:
Makes sense, thanks!